### ISSUE TITLE
[BUG] Bunkrr.su URL is Missing /V/

### ISSUE THREAD

**Issue by trdrdtrded** [Jules-WinnfieldX/CyberDropDownloader#556]
**Describe the bug**
Bunkrr videos seem to be missing "/v" in the URL path and therefore the file download is failing. Seems to only be affecting bunkrr.su
The URL should be https://bunkrr.su/v/[VIDEO] instead of https://bunkrr.su/[VIDEO].
Have not tested on other content like images. Tangentially, https://stream.bunkr.la/v/[VIDEO] seems to be working correctly.
Attached is the log of the failed download.

**To Help Reproduce**
Steps to help reproduce the behavior:
1. Try to download a video from Bunkrr. My test url can be found in the attached log.

**Setup Information**
 - OS: linux
 - Cyberdrop Downloader Version 4.2.186

**Downloader.log file**
[downloader.log](https://github.com/Jules-WinnfieldX/CyberDropDownloader/files/12410007/downloader.log)

**Comment by blackerhot** [comment]
found https://big-taco-1redir.bunkr.ru/

**Comment by Jules-WinnfieldX** [comment]
Should be fixed in 4.2.189

**Comment by CodexScript** [comment]
Not fixed as of 4.2.194
For example on the following album: **NSFW** https://bunkr.la/a/hs7coKTS
You can see that Cyberdrop doesn't capture any of the correct URLs here

```
Starting: https://bunkr.la/a/hs7coKTS
Error: https://bunkrr.su/02f1bb7ec0je84e55bf6a6553c3329c5-Xqu7iwnF.mp4
Error: https://bunkrr.su/0gm38cst759ivfll8s4cd-hUpt4pX7.mp4
Error: https://bunkrr.su/14-(1)-XuApMP1u.mp4
Error: https://bunkrr.su/15-(2)-pe9IMlW2.mp4
Error: https://bunkrr.su/9-(4)-ZtbqXh1k.mp4
Error: https://bunkrr.su/21-V3zZ1lpi.mp4
Error: https://bunkrr.su/6-(4)-e1aqDlJp.mp4
Error: https://bunkrr.su/11-HBYgnmLP.mp4
Error: https://bunkrr.su/9-(5)-DhyrGKjo.mp4
Error: https://bunkrr.su/4-(1)-U8qlmfrR.mp4
Error: https://bunkrr.su/7-(4)-xmdktjKG.mp4
Error: https://bunkrr.su/1-(5)-NjHj1Nxv.mp4
Error: https://bunkrr.su/16-apLk0LuW.mp4
Error: https://bunkrr.su/6-(10)-S1BNdVlk.mp4
Error: https://bunkrr.su/21-(1)-genesis-lopez---pool-twerk-QuRYsme0.mp4
Error: https://bunkrr.su/5-jsAzPG5Z.mp4
Error: https://bunkrr.su/2-(3)-1iZxVtzt.mp4
Error: https://bunkrr.su/12-(5)-DWRdxfQk.mp4
Error: https://bunkrr.su/3-(5)-FrmxAAfA.mp4
Error: https://bunkrr.su/6-(7)-xEUeO8Vg.mp4
Error: https://bunkrr.su/16-(3)-yXVHbV8X.mp4
Error: https://bunkrr.su/4-(10)-a3JE32hC.mp4
Error: https://bunkrr.su/3-(6)-VdzgribB.mp4
Error: https://bunkrr.su/2-(4)-SfTzU5d0.mp4
Error: https://bunkrr.su/8-vUMoZCVd.mp4
Error: https://bunkrr.su/5-(1)-(1)-NxWNkPZi.mp4
Error: https://bunkrr.su/10-(3)-CMfk1TQ1.mp4
Error: https://bunkrr.su/3-O3ljuzzC.mp4
Error: https://bunkrr.su/12-(2)-hKYvDuaT.mp4
Error: https://bunkrr.su/6-(3)-f5MPGCDp.mp4
Error: https://bunkrr.su/5-(6)-aeY4IPuN.mp4
Error: https://bunkrr.su/14-(4)-Mc0townj.mp4
Error: https://bunkrr.su/12-(4)-e34IVcuT.mp4
Error: https://bunkrr.su/8-(4)-792uBPsr.mp4
Error: https://bunkrr.su/6-(5)-cQtTPjGS.mp4
Error: https://bunkrr.su/9-(2)-S2lOppo6.mp4
Error: https://bunkrr.su/9-(1)-(1)-rJ20R8b4.mp4
Error: https://bunkrr.su/15-(3)-WV1kwl3z.mp4
Error: https://bunkrr.su/8-(1)-(1)-5qmACUc2.mp4
Error: https://bunkrr.su/10-(2)-a1bCenOO.mp4
Error: https://bunkrr.su/7-(7)-agXdvR9F.mp4
Error: https://bunkrr.su/2-SdXqG3hQ.mp4
```

**Comment by Jules-WinnfieldX** [comment]
> Not fixed as of 4.2.194 For example on the following album: **NSFW** https://bunkr.la/a/hs7coKTS You can see that Cyberdrop doesn't capture any of the correct URLs here
> 
> ```
> Starting: https://bunkr.la/a/hs7coKTS
> Error: https://bunkrr.su/02f1bb7ec0je84e55bf6a6553c3329c5-Xqu7iwnF.mp4
> Error: https://bunkrr.su/0gm38cst759ivfll8s4cd-hUpt4pX7.mp4
> Error: https://bunkrr.su/14-(1)-XuApMP1u.mp4
> Error: https://bunkrr.su/15-(2)-pe9IMlW2.mp4
> Error: https://bunkrr.su/9-(4)-ZtbqXh1k.mp4
> Error: https://bunkrr.su/21-V3zZ1lpi.mp4
> Error: https://bunkrr.su/6-(4)-e1aqDlJp.mp4
> Error: https://bunkrr.su/11-HBYgnmLP.mp4
> Error: https://bunkrr.su/9-(5)-DhyrGKjo.mp4
> Error: https://bunkrr.su/4-(1)-U8qlmfrR.mp4
> Error: https://bunkrr.su/7-(4)-xmdktjKG.mp4
> Error: https://bunkrr.su/1-(5)-NjHj1Nxv.mp4
> Error: https://bunkrr.su/16-apLk0LuW.mp4
> Error: https://bunkrr.su/6-(10)-S1BNdVlk.mp4
> Error: https://bunkrr.su/21-(1)-genesis-lopez---pool-twerk-QuRYsme0.mp4
> Error: https://bunkrr.su/5-jsAzPG5Z.mp4
> Error: https://bunkrr.su/2-(3)-1iZxVtzt.mp4
> Error: https://bunkrr.su/12-(5)-DWRdxfQk.mp4
> Error: https://bunkrr.su/3-(5)-FrmxAAfA.mp4
> Error: https://bunkrr.su/6-(7)-xEUeO8Vg.mp4
> Error: https://bunkrr.su/16-(3)-yXVHbV8X.mp4
> Error: https://bunkrr.su/4-(10)-a3JE32hC.mp4
> Error: https://bunkrr.su/3-(6)-VdzgribB.mp4
> Error: https://bunkrr.su/2-(4)-SfTzU5d0.mp4
> Error: https://bunkrr.su/8-vUMoZCVd.mp4
> Error: https://bunkrr.su/5-(1)-(1)-NxWNkPZi.mp4
> Error: https://bunkrr.su/10-(3)-CMfk1TQ1.mp4
> Error: https://bunkrr.su/3-O3ljuzzC.mp4
> Error: https://bunkrr.su/12-(2)-hKYvDuaT.mp4
> Error: https://bunkrr.su/6-(3)-f5MPGCDp.mp4
> Error: https://bunkrr.su/5-(6)-aeY4IPuN.mp4
> Error: https://bunkrr.su/14-(4)-Mc0townj.mp4
> Error: https://bunkrr.su/12-(4)-e34IVcuT.mp4
> Error: https://bunkrr.su/8-(4)-792uBPsr.mp4
> Error: https://bunkrr.su/6-(5)-cQtTPjGS.mp4
> Error: https://bunkrr.su/9-(2)-S2lOppo6.mp4
> Error: https://bunkrr.su/9-(1)-(1)-rJ20R8b4.mp4
> Error: https://bunkrr.su/15-(3)-WV1kwl3z.mp4
> Error: https://bunkrr.su/8-(1)-(1)-5qmACUc2.mp4
> Error: https://bunkrr.su/10-(2)-a1bCenOO.mp4
> Error: https://bunkrr.su/7-(7)-agXdvR9F.mp4
> Error: https://bunkrr.su/2-SdXqG3hQ.mp4
> ```

It was fixed. There is another new sub domain as of today it looks like. 4.2.195 going up now for it.
